### DISCUSSION TITLE
Threads

### DISCUSSION THREAD

**Discussion by ClosetGeek-Git** [geckosio/geckos.io#230] [General]
I was wondering if there's much knowledge out there on how libdatachannel utilizes threads within geckos? I assume that geckos doesn't (can't?) reimplement libdatachannel's IO into node's uv. If that's true it would be good to know how many threads are at work in the background, and if possible if this can be tuned.

**Comment by yandeu** [comment]
That's a question for @murat-dogan and @paullouisageneau.

**Reply by paullouisageneau** [reply]
Answered here: https://github.com/paullouisageneau/libdatachannel/issues/716#issuecomment-1323507610

**Comment by ClosetGeek-Git** [comment]
I think libdatachannel spawns 4 threads. I wonder if there are specific factors that make 4 threads optimal for each instance.
